R/build-home-index.R

```diff
@@ -100,6 +100,11 @@ data_home_sidebar <- function(pkg = ".") {
       set_names(names(components))
   )
 
+  if ("readme" %in%  pkg$meta$home$sidebar$structure
```

Comment:
```suggestion
  if ("readme" %in% pkg$meta$home$sidebar$structure
```

Reply:
But why is this handled specially to all the other components?

Reply:
Oops, because I had the navbar behavior in mind, not the sidebar one.